include/jemalloc/internal/bit_util.h

```diff
@@ -1,45 +1,33 @@
-#ifndef JEMALLOC_INTERNAL_UTIL_INLINES_H
-#define JEMALLOC_INTERNAL_UTIL_INLINES_H
-
-#ifndef JEMALLOC_ENABLE_INLINE
-unsigned	ffs_llu(unsigned long long bitmap);
-unsigned	ffs_lu(unsigned long bitmap);
-unsigned	ffs_u(unsigned bitmap);
-unsigned	ffs_zu(size_t bitmap);
-unsigned	ffs_u64(uint64_t bitmap);
-unsigned	ffs_u32(uint32_t bitmap);
-uint64_t	pow2_ceil_u64(uint64_t x);
-uint32_t	pow2_ceil_u32(uint32_t x);
-size_t	pow2_ceil_zu(size_t x);
-unsigned	lg_floor(size_t x);
-void	set_errno(int errnum);
-int	get_errno(void);
-#endif
+#ifndef JEMALLOC_INTERNAL_BIT_UTIL_H
+#define JEMALLOC_INTERNAL_BIT_UTIL_H
+
+#include "jemalloc/internal/assert.h"
 
-#if (defined(JEMALLOC_ENABLE_INLINE) || defined(JEMALLOC_UTIL_C_))
+#define BIT_UTIL_INLINE static inline
```

Comment:
I figured I'd feel you guys out on it. Having these means that if we want, we can easily force all the definitions into one .o, which could be useful if we want to e.g. put a breakpoint on one.